testhelpers/steps_app_language.go

```diff
@@ -852,9 +925,9 @@ func (ctx *TestContext) IAmPartOfTheHelperGroupOfTheThread() error {
 // to a given item.
 func (ctx *TestContext) ICanRequestHelpToTheGroupWithIDOnTheItemWithID(group, item string) error {
 	ctx.addPermissionGranted(
-		ctx.user,
 		ctx.user,
 		item,
+		"can_request_help_to",
```

Comment:
this file is becoming very long.. probably should be split during a future change...

Reply:
Yes, I was thinking about it too. It'll be split soon.